docs/tag-readme.md

```diff
@@ -0,0 +1,64 @@
+# Tagging of Repos Workflow
+
+## Purpose
+
+This workflow automates the process of creating GitHub releases by applying tags to your repositories.
```

Comment:
This workflow automates the process of creating GitHub releases by applying **tags** to your repositories.